packages/stacked_generator/test/logging/integration_test/samples/logger_disablereleasemode.dart

```diff
@@ -1,6 +1,7 @@
 import 'package:stacked_core/stacked_core.dart';
 
 @StackedApp(
+  routes: [],
```

Comment:
I don't see routes changed to required, but I just want to mention that it shouldn't be a required parameter.

Reply:
I thought someone else change it cause It shows compile error on my side, Let me double check